[improve][doc] SEO for "About" and "Get Started" (#664)
apache/pulsar-site: docs/about.md

id: about
title: Welcome to the Apache Pulsar documentation portal
sidebar_label: "About"

> description: Welcome to the Apache Pulsar documentation portal, and welcome to join the active and knowledgeable Pulsar community. Let’s begin your Pulsar journey here.

Reviewer: ```suggestion
description: Explore the Apache Pulsar documentation and join the knowledgeable Pulsar community.
```

Remove redundancy
Keep it more compact while still conveying the essential info.
